themes/bootstrap3/templates/ajax/status-full.phtml

```diff
@@ -43,6 +43,16 @@
           <span class="<?=$this->escapeHtmlAttr($statusClass)?>"><?=$this->transEsc($statusDescription, $statusDescriptionTokens)?></span>
       </td>
     </tr>
+
+    <?php foreach ($this->holdingsTextFieldNames as $holdingsTextField): ?>
```

Comment:
To get the build to pass, you'll need to run `vendor/bin/phing update-bootstrap5` to synchronize templates between the themes.